src/main/java/Problem3/Problem3.java

```diff
@@ -7,10 +7,19 @@
 import java.io.PrintStream;
 
 public class Problem3 {
-    public static void printListInReverse(ListNode list, PrintStream print) {
-        // homework
-        // !!! must use your ArrayStack or LinkedListStack in problem 1
-        // print with the following format. System.out.println() will not pass test.
-        print.println("hello world");   // place holder
+    public static void printListInReverse(ListNode list, PrintStream print)
+    {
+        if(list.next == null)
+            return;
+        LinkedListStack<Integer> stack = new LinkedListStack<>();
+        ListNode pointer = list.next;
+        while(pointer != null)
+        {
+            stack.push(pointer.val);
+            pointer = pointer.next;
+        }
+        while(stack.peek() != null)
+            print.print(stack.pop() + " ");
```

Comment:
```java
print.println(stack.pop())
```
this should work. there's an update of code in discord channels and I'm not seeing it here. why did you choose to not use it?